lib/railway_ipc/message_consumption.ex

```diff
@@ -47,7 +45,7 @@ defmodule RailwayIpc.MessageConsumption do
   end
 
   def persist_message({:ok, message_consumption}) do
-    case MessageAccess.persist_consumed_message(message_consumption) do
+    case RailwayIpc.persist_consumed_message(message_consumption) do
```

Comment:
No more `MessageAccess`. Instead these are functions on `RailwayIpc` and they call out to the `ConsumedMessage` and `PublishedMessage` contexts. Do we want to wrap these function calls in `RailwayIpc`? What is the responsibility of the `RailwayIpc` module? If its the entry point or boundary layer of the app, then I think it should only contain functions that can be called _outside_ of the hex package, i.e. by users of the library. In which case we actually want to move these persistence calls out and/or just call on `ConsumedMessage` and `PublishedMessage` contexts directly.